packages/address/src/MultisigAddress.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  bs58M,
+  bs58N,
+  bs58Version,
+  bs58MultisigPublicKey,
+  bs58NetType,
+  byteToNetType,
+  byteToKeyType,
+  sortAddresses,
+  bs58KeyType,
+} from './utils'
+import { sha256 } from 'multiformats/hashes/sha2'
+import { MULTISIG_KEY_TYPE } from './KeyTypes'
+import { NetType, MAINNET} from './NetTypes'
+import Address from './Address'
+
+export class MultisigAddress extends Address {
+  public M!: number
+
+  public N!: number
+
+  public constructor(version: number, netType: NetType, M: number, N: number, publicKey: Uint8Array) {
```

Comment:
```ESLint: This line has a length of 102. Maximum allowed is 100.(max-len)```

Reply:
@tyler-whitman - What's the accepted way to run lint on this project? I run `yarn lint` but am getting way more errors than I would expect...